examples/00-EDB/09_Configuration.py

```diff
@@ -1,7 +1,7 @@
 """
 EDB: fully configurable project
 -------------------------------
-This example shows how you can use create a project using BOM file and configuration files,
+This example shows how you can use create a project using bom file and configuration files,
```

Comment:
As indicated, this should be BOM file. It would only be lower case if a file is referenced, and then it would be in a literal string: the ``project1.bom`` file...